app/docs/0.13.x/proxy.md

```diff
@@ -1074,6 +1074,26 @@ standard HTTP proxy.
 
 [Back to TOC](#table-of-contents)
 
+### WebSocket and TLS
+
+When setting up the Service to point to your upstream WebSocket service you
+should pick the right protocol you want to use between Kong and the upstream.
+
+If you want to use TLS (`wss`) then the upstream WebSocket service must be
+defined using the `https` protocol and the matching port (usually 443). To
+connect without TLS (`ws`) the `http` protocol and port (usually 80) should be
+used.
+
+Similar to regular requests Kong will accept `ws` and `wss` connections on its
+respective `http` and `https` ports. To restrict clients to only accessing over
```

Comment:
"To enforce TLS connections from clients, set the" is probably better worded. Don't think the -ing form is correct in the current sentence of this patch.